index.md

```diff
@@ -7,51 +7,55 @@ layout: default
 ## [Equinix Amsterdam](#equinix-amsterdam)
 
 {% strip %}
-Server | Description | Stats | Last Contact
+Server | Description | System | Stats | Last Contact
 -------|-------------|-------|-------------
```

Comment:
Does this line (and the other three similar ones) not need fixing to match the headings?

Reply:
Fixed now.